### PR TITLE
Release SporTran `v1.0.0rc2`

### LINKED ISSUE #18: LAMMPS input files for examples would be helpful
I think it would be useful for users who are trying to use this package for their own input to have the LAMMPS input files corresponding to the examples provided here.  I'm trying to do exactly that, and getting terrible disagreement between my conventional autocorrelation integration analysis and the cepstrum-based output. I don't know if I'm somehow generating bad data, and if I had the LAMMPS structures and potentials (and things like the time step used for the MD), I could check if my problem is somehow inherent to my system, or merely the way I'm setting up the heat flux equilibration and sampling, for example.

### LINKED ISSUE #23: Examples: data
### LAMMPS input files
With reference to #18, LAMMPS input files would be useful.
- [x] SiO2
- [ ] NaCl

### NaCl
- [x] The physical parameters of the system (time step, volume, number of atoms, potential, LAMMPS input file) are not reported in [NaCl.dat](https://github.com/lorisercole/thermocepstrum/blob/develop/examples/data/NaCl.dat) example data.

### LINKED ISSUE #59: Generic Template of `Current`
Following the paper reviewers' comments:

- [x] write a base template class with all the mandatory methods of a current raising `NotImplementedError`.
This will insure that all the required methods would have to be implemented and not picked up from `current.Current` based class.

### PR DESCRIPTION
**Pull request by lorisercole**
Following the comments of the Computer Physics Communications' reviewer, we implemented the following changes:

- [x] Define `Current` as an abstract class (closes #59, see 902b2f61e7dc797a299e06c8e2bb19c59020991d)
  - The definition of the `_builder` method, `_current_type`, `_input_parameters`, and `_KAPPA_SI_UNITS` must be implemented in a subclass. 
  - A `GenericCurrent` (without units, where only `KAPPA_SCALE` is defined) has been defined. This takes the place of `Current`.

- [x] LAMMPS examples added (closes #18, closes #23)
  - [x] LAMMPS input/output files for silica (be12473)
  - [x] script to convert LAMMPS output file to `table` (plain text) and `dict` (numpy binary) format (be12473)
  - [x] example showing how to use the `TableFile` and `LAMMPSLogFile` classes from the `i_o` module (a7a4657)
  - [ ] LAMMPS input/output files for NaCl ? (@rikigigi)

- [x] Clarify the use of units: add appropriate definitions of the currents (formulas) in the documentation (@rikigigi)
  - [x] Delete units that are not correct or available.
- [x] Review list of requirements and package metadata.

### CODE REVIEW

File: sportran/current/units/stress.py [hunk 1]
```diff
@@ -10,51 +10,30 @@ def scale_kappa_GPa(TEMPERATURE, VOLUME):
...
-    Notice that in LAMMPS the stress is extensive, with pressure*volume units.
-    In 'real' units, this means that the stress is in atmosphere*angstrom^3.
+    Stress is computed with https://docs.lammps.org/compute_pressure.html
     INPUT:
     TEMPERATURE [K]
     VOLUME      cell VOLUME [A^3]
+    Input stress in atmospheres.
     """
-    return constants.atm**2 * 1.0e-12 / TEMPERATURE / constants.kB / VOLUME
+    return constants.atm**2 * 1.0e-12 * VOLUME / TEMPERATURE / constants.kB
```
Comment (lorisercole) [inline sportran/current/units/stress.py:27]: @fgrassel Does this look correct to you?

File: sportran/current/units/stress.py [hunk 2]
```diff
@@ -10,51 +10,30 @@ def scale_kappa_GPa(TEMPERATURE, VOLUME):
...
-    """
-    Conversion factor for the viscosity from GPUMD units to SI units.
-    Note that units for stress and pressure, derived from energy [eV] and volume [A^3],
-    are [eV/A^3].
-    INPUT:
-    TEMPERATURE [K]
-    VOLUME      cell VOLUME [A^3]
-    """
-    return constants.charge**2 * VOLUME / TEMPERATURE / constants.kB
+    return 1.0e-12 * VOLUME / TEMPERATURE / constants.kB
```
Comment (lorisercole) [inline sportran/current/units/stress.py:39]: @fgrassel same question for this one

**Comment by lorisercole** [comment]
All good, but some comments on the units. Let's wait for @fgrassel review!

File: sportran/current/units/stress.py (hunk 1 above)
Comment (fgrassel) [reply sportran/current/units/stress.py:27]: Yes, with the link to the LAMMPS documentation and the remark that the input is in atmospheres, this looks fine.

File: sportran/current/units/stress.py (hunk 2 above)
Comment (fgrassel) [reply sportran/current/units/stress.py:39]: Yes, it is fine.

**Review by fgrassel — APPROVED** [review]

**Pull request merged**